src/serial/RS232Net.hh

```diff
@@ -37,7 +37,6 @@ public:
 		} address;
 	};
 
-public:
```

Comment:
Very much a detail, personal coding style (i know other coding styles choose to do this differently).

I like to group public types, public methods, private methods, private member variables (and typically there shouldn't be any public member variables) together, separated by public/private keywords. Even if that means repeating `public` or `private` when the previous section had already the same access specifier.